test/resolvers_test.go

```diff
@@ -679,17 +679,28 @@ func setupGitea(ctx context.Context, t *testing.T, c *clients, namespace string)
 	}
 
 	// Sleep 5 seconds to make sure the pod gets created, then wait for it to be running. It'll take over 30s, due to
-	// waiting for the postgres and memcached pods it depends on to be running.
+	// setting up of cache and db.
 	time.Sleep(5 * time.Second)
-	if err := WaitForPodState(ctx, c, "gitea-0", namespace, func(r *corev1.Pod) (bool, error) {
+	labelSelector := "app.kubernetes.io/name=gitea,app.kubernetes.io/instance=gitea"
```

Comment:
what about dropping all of that and just use a shell script instead ? ie with installing forgejo with helm  https://github.com/chmouel/startpaac/tree/main/lib/forgejo